modules/serverless-functions-specifying-function-revision.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-on-cluster-builds.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-functions-specifying-function-revision_{context}"]
+= Specifying function revision
+
+When building and deploying a function on the cluster as opposed to locally, you need to point to the correct function revision by specifying the Git repository, branch, location within the repository, and other parameters. In addition to specifying these parameters using the `func.yaml` configuration file, you can use the flags for the `kn func deploy` command.
```

Comment:
```suggestion
When building and deploying a function on the cluster, you must point to the correct function revision, by specifying the Git repository, branch, and location within the repository. You can specify these parameters in the `func.yaml` configuration file, or by using flags with the `kn func deploy` command.
```

Reply:
I agree with these suggestions, but also, I would make some additional changes.
```suggestion
When building and deploying a function on the cluster as opposed to locally, you need must specify the location of your code. This includes the Git repository URL, the branch if not the main branch, and any subdirectory within the repository if your function is not at the root level. You can specify these parameters in the `func.yaml` configuration file, or by using flags with the `kn func deploy` command.
```
The meaning is just slightly different. You don't specify a "revision", but rather a location. Additionally, the subdirectory and branch are not required.